BLSQ/iaso
# IA-2924: Comment title is the same for rejection and approval

Explain what problem this PR is resolving
- Comment title is the same for rejection and approval
Related JIRA tickets : [IA-2924](https://bluesquare.atlassian.net/browse/IA-2924)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Added the partial approval title message
- display different message according to that it's a rejection or a partial approval

## How to test
- Go on the org unit review changes requests
- Try to reject a change or to partial approve a change
- check which title is displayed on the comment dialog

## Print screen / video
[Screencast from 2024-08-08 11-14-22.webm](https://github.com/user-attachments/assets/d046b4c6-44ca-41ca-96dd-5ef2f47c9546)


[IA-2924]: https://bluesquare.atlassian.net/browse/IA-2924?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
